### ISSUE TITLE
Script 5: Bring the Analysis Together

### ISSUE THREAD

**Issue by rtaph** [UBC-MDS/bc_election_turnout#28]
A fifth script: an .Rmd or .ipynb files that presents the key useful (not all!) exploratory data analysis as well as the statistical summaries and figures in a little report. There should be a written narrative in this document that introduces and justifies your question, introduces the data set, presents the findings/results, and interprets the findings/results in context of the question. Some critique of the analysis is also expected (limitations, assumptions, etc) and a statement of future directions (what would you do next if you had more time to work on this). The report is expected to be 1-2 written pages (excluding figures, tables and references). You are expected to have a reference section and cite 2-3 external sources (data source can be one of these citations) in addition to citing the programming languages and packages used. Yes, you need to cite the programming languages and packages used in your project. You will learn how to do this in lecture.

**Comment by ChadNeald** [comment]
Hey everyone. I checked the example milestone 2 project and it had the following sections so I made this checklist to keep track of what's done and who is working on what. I'm starting on the Data section. I also think the majority of what we have written already can be used for the intro section maybe with just a quick review of it. Feel free to add any sections I've missed!

Completed Sections
- [x] Summary
- [x] Introduction
- [x] Data
- [x] Analysis
- [x] Results and Discussion
- [x] References

**Comment by ChadNeald** [comment]
I'll get the skeleton project report set up and push it after 1 (I'm heading to that alumni talk)

**Comment by ChadNeald** [comment]
Ok the skeleton version is up right now with the different section headers placed in. I moved the intro from the readme into the report as well but I couldn't get the images to show up so I've commented them out in the report for now. Not sure what I'm doing wrong maybe others will have better luck. But we can start with this skeleton and add to it piece by piece.

**Comment by yuanxiongbear** [comment]
I uploaded a partial script 5 onto my fork. And for your review this is the link:
https://github.com/yuanxiongbear/bc_election_turnout/blob/script3/src/script5_draft.Rmd

In the script I covered the plots part (plot description and brief conclusion of the plot). The format I used is referred to the count_report.Rmd file in data_analysis_pipline example of Tiffany's. 

I think if we OK with it I can just copy the code into our final script5. 

cheers
